dds_cli/base.py

```diff
@@ -70,13 +71,17 @@ def __init__(
 
             # Use user defined destination if any specified
             if self.method in DDS_DIR_REQUIRED_METHODS:
-                self.dds_directory = dds_cli.directory.DDSDirectory(
-                    path=dds_directory
-                    if dds_directory
-                    else pathlib.Path.cwd()
-                    / pathlib.Path(f"DataDelivery_{dds_cli.timestamp.TimeStamp().timestamp}")
+                default_dir = pathlib.Path(
+                    f"DataDelivery_{dds_cli.timestamp.TimeStamp().timestamp}"
                 )
-
+                if mount_dir:
+                    new_directory = mount_dir / default_dir
```

Comment:
I can't remember now if the CLI has config via environment variables etc, but it could be a nice addition if it does.

I feel like we will have quite a lot of flags when running at the NGI that we will need to remember each time. We can make aliases / wrappers of course though.

Reply:
Wait, to append a new directory to a path with `pathlib` you use the `/` operator? Cool! 😆

Reply:
As an alternative to environment vars, could also look for a YAML config file in the same directory as where we store the token etc too.. But yeah, low priority. And as it's probably only us and we'll probably wrap this in a wrapper tool anyway it's mega low priority. Enough that we can just forget about it I think 😅

Reply:
Okey 😄 👍🏻

Reply:
![tucool](https://c.tenor.com/xWbrE4xROXkAAAAd/thumbs-up-cool.gif)